### PR TITLE
Bug 1137593 - Use notifcation options.dir instead of options.bidi

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1137593

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8f18b1307b2e5af9163343529b27304b7488237d

File: apps/system/js/notifications.js [hunk 1]
```diff
@@ -431,7 +431,7 @@ var NotificationScreen = {
     notificationNode.dataset.notificationId = detail.id;
     notificationNode.dataset.noClear = behavior.noclear ? 'true' : 'false';
     notificationNode.lang = detail.lang;
-    notificationNode.dataset.predefinedDir = detail.bidi;
```
Comment (mikehenrty) [inline apps/system/js/notifications.js]: Make sure you also change the `detail.bidi` below:
https://github.com/mozilla-b2g/gaia/pull/29390/files#diff-9a74bae315146783bc7b24d5d7302a95R493

Comment (mikehenrty) [inline apps/system/js/notifications.js]: Ah, that link didn't work. Line 493 was what I was referring to.
```
oldNotif.dataset.predefinedDir = detail.bidi;
```

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6664a8bc35fa71d32f29b5c5ff52f885d8588c72

File: apps/system/js/notifications.js (hunk 1 above)
Comment (zbraniecki) [inline apps/system/js/notifications.js]: thanks!

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2334013f9f921e0bc5d951b0e96a6b3b033f5e08

**Pull request merged**